components/public-api/gitpod/experimental/v1/ide_workspace.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package gitpod.experimental.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1";
+
+service IDEWorkspaceService {
+    // SendHeartbeat sends a heartbeat signal to a running workspace.
+    rpc SendHeartbeat(SendHeartbeatRequest) returns (SendHeartbeatResponse) {}
+
+    // SendCloseSignal sends a close signal to a running workspace.
+    rpc SendCloseSignal(SendCloseSignalRequest) returns (SendCloseSignalResponse) {}
```

Comment:
`Close` May be sufficient, without leaking implementation details (it being a signal)